File: exercise/sum/solution/correct.js
const getallen = process.argv.slice(1)
let som = 0

for (const getal of getallen) {
    const r = parseInt(getal)
    if (isNaN(r)) {
        console.error("som: ongeldige argumenten")
        process.exit(1);
    } else {
        som += r
    }
}

console.log(som);

File: tests/exercises/sum/solution/correct.cs
int som = 0;
foreach (string arg in args)
{
  try
  {
    som += Int32.Parse(arg);
  }
  catch (FormatException)
  {
    Console.Error.WriteLine("som: ongeldige argumenten");
    Environment.Exit(1);
  }
}
Console.WriteLine(som);
